<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>book of household</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QFormLayout" name="formLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_name">
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_art">
         <property name="text">
          <string>Art:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QTextEdit" name="textEdit_discription"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_summe">
         <property name="text">
          <string>Summe:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_summe"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_kategorie">
         <property name="text">
          <string>Kategorie:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="comboBox_categorie"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_Date">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="dateTime">
          <datetime>
           <hour>8</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2012</year>
           <month>7</month>
           <day>10</day>
          </datetime>
         </property>
         <property name="date">
          <date>
           <year>2012</year>
           <month>7</month>
           <day>10</day>
          </date>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="comboBox_art">
         <item>
          <property name="text">
           <string>Eingaben</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ausgaben</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_beschreibung">
         <property name="text">
          <string>Beschreibung:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_submit">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_close">
         <property name="text">
          <string>close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>433</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <widget class="QMenu" name="menuCategorie">
     <property name="title">
      <string>Category</string>
     </property>
     <addaction name="actionAdd_Categorie"/>
     <addaction name="actionDelete_Categorie"/>
    </widget>
    <addaction name="menuCategorie"/>
    <addaction name="actionShow_Input"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAdd_Categorie">
   <property name="text">
    <string>Add Categorie</string>
   </property>
  </action>
  <action name="actionDelete_Categorie">
   <property name="text">
    <string>Delete Category</string>
   </property>
  </action>
  <action name="actionShow_Input">
   <property name="text">
    <string>Show Input</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>doubleSpinBox_summe</tabstop>
  <tabstop>comboBox_categorie</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>comboBox_art</tabstop>
  <tabstop>textEdit_discription</tabstop>
  <tabstop>pushButton_submit</tabstop>
  <tabstop>pushButton_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
